src/pages/MonthlyGood/MonthlyGoodThanksPage.vue

```diff
@@ -164,12 +164,13 @@ export default {
 @import 'settings';
 
 .monthly-good-thanks-page {
-	padding-top: 1rem;
+	padding-top: 4rem;
 
 	h1 { margin-bottom: 1rem; }
 
 	.icon-confirmation {
 		height: 3rem;
+		width: 3rem;
```

Comment:
This icon was broken, not sure when this happened. Also added a little bit of padding top. This thanks page and the AD thanks page are kind of boring and could use a redesign.